### PR TITLE
use strpos for fixed string matching

### PR DESCRIPTION
**Pull request by glensc**
refs 8d18478

### CODE REVIEW

**Comment by glensc** [comment]
travis build failure is not related to this PR

**Comment by vaclavvanik** [comment]
Can u add test?

**Comment by Ocramius** [comment]
@vaclavvanik it's an idempotent change

File: src/Transport/Sendmail.php [hunk 1]
```diff
@@ -230,7 +230,7 @@ protected function prepareHeaders(Mail\Message $message)
         $from = $headers->get('From');
         if ($from) {
             foreach ($from->getAddressList() as $address) {
-                if (preg_match('/\\\"/', $address->getEmail())) {
+                if (strpos($address->getEmail(), '/\\\"/') !== false) {
```
Comment (Ocramius) [inline src/Transport/Sendmail.php]: This looks wrong - you are using the regex as a `strpos()` lookup? :-\

**Comment by glensc** [comment]
this test passes now

```
➔ phpunit test/Transport/SendmailTest.php
PHPUnit 4.8.35 by Sebastian Bergmann and contributors.

.SS...

Time: 144 ms, Memory: 6.75MB

OK, but incomplete, skipped, or risky tests!
Tests: 6, Assertions: 13, Skipped: 2.
```

**Comment by glensc** [comment]
the requested changes are made. travis fault is not mine. 
@Ocramius your review change is till pending, accept these.

**Comment by glensc** [comment]
@weierophinney this is complete now

**Comment by Ocramius** [comment]
@glensc sorry for the long wait - I manually merged this via zendframework/zend-mail@a6f72a1926d35e0dfadbaa0b12dc1b563b825a7a

Thanks!

**Pull request closed without merging**